rest-api/calamari_rest/serializers/v2.py

```diff
@@ -142,6 +142,8 @@ class Meta:
     public_addr = serializers.CharField(read_only=True, help_text="Public/frontend IP address")
     cluster_addr = serializers.CharField(read_only=True, help_text="Cluster/backend IP address")
     crush_node_ancestry = serializers.Field(help_text="An ordered list of CRUSH node ids that represent a path from the parent node of this OSD up to the root of the tree")
+    backend_partition_path = serializers.CharField(read_only=True, help_text="Full path to the storage partition targeted by this OSD")
```

Comment:
@joehandzik What provision do you make for cases where an OSD has it's journal and data on different devices?